endpoints/v2/__init__.py

```diff
@@ -61,8 +63,19 @@ def handle_readonly(ex):
 
 
 @v2_bp.app_errorhandler(UpstreamRegistryError)
-def handle_proxy_cache_errors(error):
+def handle_proxy_cache_error(error):
```

Comment:
This and the following function are very similar perhaps make the response formatting a function and reuse that way.

Reply:
good point - done